src/webview/src/components/fbaCheckbox.js

```diff
@@ -39,12 +49,19 @@ import DataProviderEditDialog from './dataProviderEditDialog';
 //  - use Chips instead of texts (allowing always to set the <DoneIcon />?)
 // - highlight current selection with a different text. e.g. "keep as OK", ...
 // - add id= to buttons...
-// - use e.g. react-markdown to support markdown for background, desc, comments
 
 function Alert(props) {
     return <MuiAlert elevation={6} variant="filled" {...props} />;
 }
 
+function GetTextValue(property) {
+    if (property && property.textValue) {
+        return property.textValue;
+    }
+
+    return '';
```

Comment:
to avoid the break to former .fba files you could check whether typeof property === 'string' and return that value, or?
(or we do the "once" conversion in the fbaEditor.ts:
- increase fba version to 0.3
- add convertV02Texts function (similar to convertV01Effects) to convert once and then mark the doc version as 0.3